src/components/property-page-widgets/HostProfile/utils/getDescription.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { size } from 'lodash';
+
+import { GridColumn } from 'layout/GridColumn';
+import { Paragraph } from 'typography/Paragraph';
+
+/**
+ * @param {string} description
+ * @return {Object}
```

Comment:
This should be

```suggestion
 * @return {Object|null}
```

You'll also have to define null so the top line of the JSdoc should be 

` * @typedef {Object} null`

Check `Dropdown/utils/getDefaultValue.js`